apidoc/Titanium/Calendar/Event.yml

```diff
@@ -249,9 +250,13 @@ properties:
     
   - name: recurrenceRules
     summary: The recurrence rules for the calendar item. (Available in iOS 5.1 and above.)
+    description: |
+        On Android only the first element of the recurrenceRules is take into account 
+        due to the way it handles conditions for recurrence rules.
```

Comment:
We should add a comma after "On Android," and the word "take" should be "taken".

```
On Android, only the first element of the recurrenceRules is taken into account
due to the way it handles conditions for recurrence rules.
```